docs/setup/settings.asciidoc

```diff
@@ -304,7 +304,7 @@ list of timezones, refer to https://en.wikipedia.org/wiki/List_of_tz_database_ti
 
 | [[logging-verbose]] `logging.verbose:` {ece-icon}
  | Set to `true` to log all events, including system usage information and all
-requests. *Default: `false`*
+requests. Requires `logging.silent` and `logging.quiet` set to `false`. *Default: `false`*
```

Comment:
Would it also be worth adding a note to `logging.quiet` that it requires `logging.silent` to be `true`?

Reply:
Maybe I can be clearer here, but if either `logging.silent` or `logging.quiet` are set to `true` then also setting `logging.verbose` to `true` will result in Kibana failing to start. Put another way, the `logging.verbose` setting is mutually exclusive to these other two settings.
Separately, I don't think if `logging.silent` is `true` then `logging.quiet` also needs to be `true` or vice-versa to avoid issues, but I can also test this in case it's another dependency.
We might indicate the opposite configuration in the docs for these other two settings if you want to use them then you need to ensure verbose is false, which should cover warning for this startup failure scenario from the context of these two different groups of settings.